doc/source/examples/dem/rectangular-hopper/rectangular-hopper.rst

```diff
@@ -0,0 +1,210 @@
+        set inserted number of particles at each time step = 2910
+        set insertion frequency                            = 25000
+        set insertion box minimum x                        = -0.1030
+        set insertion box minimum y                        = 0.10644
+        set insertion box minimum z                        = .00224
+        set insertion box maximum x                        = 0.1030
+        set insertion box maximum y                        = 0.16020
+        set insertion box maximum z                        = 0.03136
+        set insertion distance threshold                   = 1.5
+        set insertion random number range                  = 0.1
+        set insertion random number seed                   = 20
+    end
+
+
+Lagrangian physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The total number of particles in this simulation is 40740 with a diameter of 2.24 mm.
+
+.. code-block:: text
+
+    subsection lagrangian physical properties
+        set gx                       = 0.0
+        set gy                       = -9.81
+        set gz                       = 0.0
+        set number of particle types = 1
+        subsection particle type 0
+            set size distribution type            = uniform
+            set diameter                          = 0.00224
+            set number                            = 40740
+            set density particles                 = 2500
+            set young modulus particles           = 1000000
+            set poisson ratio particles           = 0.3
+            set restitution coefficient particles = 0.94
+            set friction coefficient particles    = 0.2
+            set rolling friction particles        = 0.09
+        end
+        set young modulus wall           = 1000000
+        set poisson ratio wall           = 0.3
+        set friction coefficient wall    = 0.2
+        set restitution coefficient wall = 0.9
+        set rolling friction wall        = 0.09
+    end
+
+
+Model parameters
+~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection model parameters
+        set contact detection method                = dynamic
+        set dynamic contact search size coefficient = 0.9
+        set load balance method                     = frequent
+        set load balance frequency                  = 200000
+        set neighborhood threshold                  = 1.3
+        set particle particle contact force method  = hertz_mindlin_limit_overlap
+        set particle wall contact force method      = nonlinear
+        set integration method                      = velocity_verlet
+    end
+
+
+Simulation control
+~~~~~~~~~~~~~~~~~~
+
+.. code-block:: text
+
+    subsection simulation control
+        set time step        = 1e-5
+        set time end         = 7.5
+        set log frequency    = 1000
+        set output frequency = 1000
+        set output path      = ./output/
```

Comment:
Can you make sure the folder is pre-generated with a dummy file?
The automatic folder generation does not work for all OSes (it generally does but still)

Reply:
How? Is it done with in example?

Reply:
you just make a file dummy in the output folder. Just a file with a random line it in. For example :
https://github.com/lethe-cfd/lethe/blob/master/examples/incompressible-flow/3d-nitsche-mixer-with-pbt-impeller/output/placeholder